agentmanigest.yaml

```diff
@@ -0,0 +1,91 @@
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: preflight
+---
+apiVersion: v1
+kind: Secret
+metadata:
+  name: agent-config
+  namespace: preflight
+type: Opaque
+stringData:
+  config.yaml: |-
+    schedule: "* * * * *"
+    token: "###AUTH_TOKEN###"
+    endpoint:
+      protocol: https
+      host: preflight.jetstack.io
+      path: /api/v1/datareadings
+    data-gatherers:
+    - name: "k8s/pods"
+      kind: "k8s"
+      config:
+        resource-type:
+          resource: pods
+          version: v1
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: agent
```

Comment:
There is a reference to this service account here: https://github.com/jetstack/preflight/blame/master/docs/datagatherers/gke.md#L104

Now seems as good a time as any to correct it from `default`